AndelaOSP/andela-societies-frontend
# Resolves [157426670] Add ability to log activities

[Pivotal Tracker Story](https://www.pivotaltracker.com/story/show/157426670)

### Description
- Implement the redux flow to log an activity
- Display a snackbar with request progress notification

### How to test
- Logged in as a fellow, click on the FAB to open a modal with the log activity for
- Fill in the form selecting a date, category and type in a description for an activity and click on the Log button
- You should see a notification on whether the request was successful or if there was an error

### Screenshots
Validation
<img width="1439" alt="screen shot 2018-05-28 at 16 09 55" src="https://user-images.githubusercontent.com/13675851/40615953-adcc4864-6291-11e8-8ec7-211a90388f37.png">

Progress
<img width="1440" alt="screen shot 2018-05-28 at 16 07 11 1" src="https://user-images.githubusercontent.com/13675851/40615974-ccfb4942-6291-11e8-8214-4bc7e9549c86.png">

Success
<img width="1440" alt="screen shot 2018-05-28 at 16 07 11 3" src="https://user-images.githubusercontent.com/13675851/40615994-dd652712-6291-11e8-8c03-94bce57dc7cc.png">

Error
<img width="1440" alt="screen shot 2018-05-28 at 16 09 28" src="https://user-images.githubusercontent.com/13675851/40616006-e8ac93e4-6291-11e8-9806-16356b2a51ce.png">
